index.html

```diff
@@ -201,7 +209,7 @@ <h2 class="content-h2">Contact me</h2>
                 <textarea id="msg" placeholder="write a message here..." name="user_message" maxlength = "500" required></textarea>
               </li>
           <li>
-            <button class="in-touch">Get in touch</button>
+            <button type="submit" value="Submit" class="in-touch">Get in touch</button>
           </li>
             </ul>
           </form>
```

Comment:
here we need to make this section flexbox  and give it a margin to prevent the header from taking the full width , the background is not well positioned as well , and finally please use the same placeholder color 

![figma contact](https://user-images.githubusercontent.com/59707859/230376403-0f2818e9-61b7-45eb-a387-5a84e7fe14c4.PNG)

![student contact](https://user-images.githubusercontent.com/59707859/230376438-881258b4-2e2c-40d5-84cb-fbe0b4dbf0f3.PNG)

Reply:
@souad988 Thank you very much, kindly assist me with the placeholder color code and the header background color.